ocsp/responder.go

```diff
@@ -195,10 +202,10 @@ func (rs Responder) ServeHTTP(response http.ResponseWriter, request *http.Reques
 	now := rs.clk.Now()
 	if now.Before(parsedResponse.NextUpdate) {
 		expiresIn := int(parsedResponse.NextUpdate.Sub(now) / time.Second)
-		maxAge = &expiresIn
+		addCacheHeader(response, &expiresIn)
 	} else {
-		zero := 0      // XXX: we want max-age=0 but since this is technically an authorized OCSP response
-		maxAge = &zero //      (despite being stale) and 5019 forbids attaching no-cache we get a little tricky
+		zero := 0                       // XXX: we want max-age=0 but since this is technically an authorized OCSP response
```

Comment:
At least in Boulder, we use XXX to indicate "Don't merge this," and TODO to indicates long-term TODO's in merged code.